docs/docs/operator/operatorRuntime.md

```diff
@@ -0,0 +1,86 @@
+---
+id: operator-runtime-mermaid
+title: Operator Runtime Flow
+sidebar_label: Operator Runtime Flow
+sidebar_position: 3
+---
+
+# Operator Runtime Flow Diagram
+
+This diagram illustrates the flow and interactions within the `operatorRuntime` function and its related functions in the Sentry Core package.
+
+```mermaid
+graph TD;
+    operatorRuntime --> getSubgraphHealthStatus
+    operatorRuntime <--> fetchBlockNumber
```

Comment:
What happens when the sub-graph is not healthy here? 

RPC call flows missing from the diagram.

Reply:
This should be updated and add the case for the subgraph beeing unhealthy.